svelte-development/src/MapPoints.svelte

```diff
@@ -45,7 +61,15 @@
   }
 
   function handleMouseOver() {
-    select(this).attr('fill-opacity', 1)
+    select(this).attr('fill-opacity', 1);
+
+    let hoverRegionIndex = select(this).attr('data-region-index');
+    hoverRegionCode = regions[hoverRegionIndex].code;
+
+    let links = regionFlow.filter(function(d) {
```

Comment:
@Yuriko-Schumacher I tried to add the links to the map, but I got stuck since I'm not as familiar how you set-up the force-directed graph.